src/OSPSuite.R/Services/DimensionTask.cs

```diff
@@ -73,6 +85,44 @@ public double[] ConvertToUnit(string dimensionName, string targetUnit, double va
 
       public string[] AllAvailableDimensionNames() => _dimensionFactory.DimensionNames.OrderBy(x => x).ToArray();
 
+      public IDimension CreateUserDefinedDimension(string unit)
+      {
+         return new Dimension(Constants.Dimension.NO_DIMENSION.BaseRepresentation, unit, unit);
+      }
+
+      public IDimension DimensionForStandardPKParameter(int standardPKParameterValue)
```

Comment:
The idea here is to be able to retrieve the deafult dimenson for a user defined pk parameter just based on the base paraemter used. e.g. AUC t4 t5 wil have the same dim as AUC automatically

Reply:
How should we deal with normalized parameters (XXX/Dose and XXX/Dose/BW)?
Theoretically in this case the dimension must be set by user along with the normalization factor?

Reply:
I think we need to add a _norm so that the default dimension is retrieve automatically
Otherwise the user can specify a unit. THe core will try to find the dimension that matches the unit. If this is not successful, then a dimension will be created with only ONE unit. I am not sure what cases we will hvae but this should cover most of them... Let see during our tests